### ISSUE TITLE
[mtproto-storage-fs] Unexpected end of JSON input

### ISSUE THREAD

**Issue by irudoy** [zerobias/telegram-mtproto#77]
```
SyntaxError: ./storage.json: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /.../node_modules/jsonfile/index.js:36:18
    at /.../node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:437:3)
```
It happens, when several clients use the same storage.

**Comment by goodmind** [comment]
This is fixed in last version, but this version not in npm still

**Comment by irudoy** [comment]
I am glad that this version, at least, exists (wherever it is) 😄

**Comment by zerobias** [comment]
@irudoy fixed version now published in npm, try `mtproto-storage-fs@0.2.0`
